client/js/angular/services/Sessions.service.js

```diff
@@ -62,8 +62,9 @@
                         if (active_session) {
                             $location.path('/session/' + active_session.uuid);
                         }
+                    } else {
+                        $location.path('/welcome');
```

Comment:
ohhhhhhh lol, i was trying to see what was up in the code, missed this completely